ihontaryk/homework_7/retrying.py

```diff
@@ -0,0 +1,30 @@
+import random
+
+
+def retry(func):
+    """
+    Decorator which will retry inner function if it failed
+    """
+
+    def wrapper(*args, **kwargs):
+        while True:
+            try:
+                return func(*args, **kwargs)
+            except Exception as e:
+                print(e)
+                continue
+
+    return wrapper
```

Comment:
create also parametrised decorator which would receive as parameter number of tries, if nothing is passed - infinit amount of tries
cover with tests

Reply:
Created decorator with parameters. Tests will be added in separate PR